funnel/templates/proposal.html.jinja2

```diff
@@ -121,26 +121,16 @@
           <iframe class="facebooklike" src="//www.facebook.com/plugins/like.php?app_id=114496105304651&amp;href={{ request.url }}&amp;send=false&amp;layout=button_count&amp;width=150&amp;show_faces=false&amp;action=recommend&amp;colorscheme=light&amp;font&amp;height=21" scrolling="no" frameborder="0" style="border:none; overflow:hidden; width:150px; height:21px;" allowTransparency="true"></iframe>
         </div>
         <div class="col-md-6 col-lg-6 col-sm-12 col-xs-12">
-          Status:
-          {%- if 'view-status' in g.permissions %}
-            {{ proposal.state.label.title }}
-            {% if proposal.session and proposal.state.CONFIRMED %} &amp; Scheduled{% endif %}
-          {%- else %}
-            {% if proposal.state.CONFIRMED %}
-              Confirmed {% if proposal.session %} &amp; Scheduled{% endif %}
-            {% else %}
-              Submitted
-            {% endif %}
-          {% endif %}
-          {% if proposal.session %}
-            <a class="btn btn-xs btn-info" href="{{ proposal.session.url_for() }}">View session in schedule</a>
-          {% endif %}
-          {%- if 'move-proposal' in g.permissions %}
-            <br>
-            {{ renderform(proposal_move_form, 'proposal-move', action=proposal.url_for('move-to'), submit='Move', style=None) }}
-          {%- endif %}
+          Status: {% if proposal.state.SCHEDULED %}{{ proposal.state.SCHEDULED.label }}{% elif proposal.state.CONFIRMED %}{{ proposal.state.CONFIRMED.label }}{% else %}Submitted{% endif %}
```

Comment:
This can be reduced to `{% if proposal.state.CONFIRMED %}{{ proposal.state.label }}{% else %}{% trans %}Submitted{% endtrans %}{% endif %}`